src/Settings/BankingAccountTypeSettings/BankingAccountTypeSettings.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesShape } from '@folio/stripes/core';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import { getControlledVocabTranslations } from '@folio/stripes-acq-components';
+
+import { BANKING_ACCOUNT_TYPES_API } from '../constants';
+
+class BankingAccountTypeSettings extends React.Component {
```

Comment:
Please, refactor to use the functional component instead of the class one. We don't use _class components_ in the new code.